docs/api-docs/storefront/widgets/widgets-tutorial.md

```diff
@@ -35,36 +39,35 @@ In `templates/pages/category.html`, add `{{{region name="category_header_banner"
 <!-- ... -->
 ```
 
-If you're using [Stencil CLI](https://developer.bigcommerce.com/stencil-docs/installing-stencil-cli/installing-stencil) and editing theme files locally, push and apply your changes before proceeding to the next step.
+If you’re using [Stencil CLI](https://developer.bigcommerce.com/stencil-docs/installing-stencil-cli/installing-stencil) and editing theme files locally, push and apply your changes before proceeding to the next step.
 
 <div class="HubBlock--callout">
 <div class="CalloutBlock--info">
 <div class="HubBlock-content">
 
 > ### Note
 > * Regions can only be added to templates in [`templates/pages/`](https://developer.bigcommerce.com/stencil-docs/storefront-customization/directory-structure).
-> * [Install and Configure Stencil CLI](https://developer.bigcommerce.com/stencil-docs/installing-stencil-cli/installing-stencil) to edit theme files locally.
-> * Use [Page Builder](https://support.bigcommerce.com/s/article/Stencil-Themes#edit) to edit theme files via control panel UI.
+>* To edit theme files locally, use [Stencil CLI](https://developer.bigcommerce.com/stencil-docs/installing-stencil-cli/installing-stencil).
+> * To edit theme files in the control panel, use [Page Builder](https://support.bigcommerce.com/s/article/Stencil-Themes#edit).
 
 </div>
 </div>
 </div>
 
-## Verify region creation
+### Verify region creation
 
-To verify region creation, send a `GET` request to [`/v3/content/regions`](https://developer.bigcommerce.com/api-reference/storefront/widgets-api/regions/getcontentregions).
+To verify region creation, send a `GET` request to [`/v3/content/regions`](https://developer.bigcommerce.com/api-reference/storefront/widgets-api/regions/getcontentregions) using `?template_file=pages/category` query string parameter.
 
 ```http
 GET /stores/{{STORE_HASH}}/content/regions?template_file=pages/category
 Host: api.bigcommerce.com
 X-Auth-Token: {{ACCESS_TOKEN}}
-X-Auth-Client: {{CLIENT_ID}}
 Accept: application/json
 ```
 
 [![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](https://developer.bigcommerce.com/api-reference/storefront/widgets-api/regions/getcontentregions#requestrunner)

```

Comment:
Should the request runner include a query value for the template file? template_file is a required parameter.

Reply:
At this time, we cannot add query parameters to the Request Runner. I will add a note about including this query parameter value.